doc/decisions/3_decided/misc_namespace.md

```diff
@@ -0,0 +1,65 @@
+# Namespace for miscellaneous data
+
+## Problem
+
+A `KeySet` is powerful data structure, which could be use for data other than configuration data.
+Especially with the hashmap implementation, using a `KeySet` for generic data can be useful.
+For this generic data we do not care about the namespace-related features.
+
+However, because a `Key` cannot exist without a namespace, we need to choose a namespace.
+
+## Constraints
+
+- The outcome must be compatible with the ["Types of `KeySet`s" decision](../2_in_progress/keyset_types.md).
+
+## Assumptions
+
+- The ["Types of `KeySet`s" decision](../2_in_progress/keyset_types.md) leads to separate types of `KeySet`s that are restricted w.r.t. the namespaces they may contain.
+
+## Considered Alternatives
+
+### Use cascading `Key`s
+
+We could use the existing `KEY_NS_CASCADING` for one of the types of `KeySet`s.
+
+This may be confusing to users of the APIs.
+The purpose of `KEY_NS_CASCADING` becomes muddied.
+It is no longer just for lookup, but would now have a secondary purpose.
+
+### Introduce separate `KEY_NS_MISC` namespace
+
+We introduce a new `KEY_NS_MISC`, which is used exclusively for generic and miscellaneous data.
+It implies a separate [type of `KeySet`](../2_in_progress/keyset_types.md).
+
+This leaves `KEY_NS_CASCADING` for its original purpose, while still solving the issue with generic data.
+
+## Decision
+
+Introduce separate `KEY_NS_MISC` namespace.
+
+In the unescaped form this will be `\x09` (byte with decimal value `9`).
+In the escaped form it will be `data:/`.
```

Comment:
Is there a reason why the escaped form is not `misc:/`?

Reply:
Good catch, seems I forgot to change it

Reply:
```suggestion
In the escaped form it will be `misc:/`.
```